databases/week2/part4clubDatabase.sql

```diff
@@ -0,0 +1,46 @@
+-- Part 4: Creating a club database
+
+CREATE DATABASE football_club;
+USE football_club;
+
+-- Creating tables
+SET NAMES utf8mb4;
+-- Players
+CREATE TABLE `player` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `age` int(3) NOT NULL,
+  `game_id` int(10) unsigned NOT NULL,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `fk_game_id` FOREIGN KEY (`game_id`) REFERENCES `game` (`id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- Game
+CREATE TABLE `game` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `played` DATETIME NULL DEFAULT NULL,
+  `result` varchar(255),
+  `player_id` int(10) unsigned NOT NULL,
+  PRIMARY KEY (`id`),
+  CONSTRAINT `fk_player_id` FOREIGN KEY (`player_id`) REFERENCES `player` (`id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Comment:
I'm getting an error from these two `Error in query (1824): Failed to open the referenced table 'game'` and `Error in query (1824): Failed to open the referenced table 'player'`

Can you guess why?